### DISCUSSION TITLE
Dispatch mouse scroll events

### DISCUSSION THREAD

**Discussion by wealthr** [kaliiiiiiiiii/Selenium-Driverless#72] [Ideas]
HOW CAN I EMULATE REAL MOUSE SCROLLING OF THE PAGE?

**Comment by ttlns** [comment]
That isn't yet implemented

**Comment by wealthr** [comment]
alright, would you please add it to your todo?, would be nice having it implemented

**Reply by ttlns** [reply]
aceually, it allready is here: https://github.com/kaliiiiiiiiii/Selenium-Driverless#todos

I'll have to analyze how "human" scrolling behaves and randomize it:)
